_includes/connect-zbt-1/troubleshooting/connect-zbt-is-not-discovered.md

```diff
@@ -0,0 +1,14 @@
+## Symptom
+
+I plugged in {{ productName }} but the {{ productName }} integration does not show up on the integration page. It is not in the **Discovered** section.
+
+## Resolution
+
+You can try the following steps:
+
+1. Try a different USB port.
+2. Refresh your browser page.
+3. Restart Home Assistant to resolve missing USB discoveries.
+4. Make sure the {{ productNameShort }} integration is not hidden. Select **Show ignored integrations**.
```

Comment:
productNameShort does not exist.

Reply:
oh! Thank you for the catch! 
https://github.com/NabuCasa/support/pull/742